isic/core/serializers.py

```diff
@@ -1,12 +1,29 @@
 from rest_framework import serializers
+from rest_framework.fields import ListField
 
 from isic.core.models import Image
 from isic.core.models.image import RESTRICTED_SEARCH_FIELDS
 
 
+class StringArrayField(ListField):
+    def to_representation(self, obj):
+        obj = super().to_representation(obj)
+        return ','.join([str(element) for element in obj])
+
+    def to_internal_value(self, data):
+        if data:
+            # if the data is coming from swagger, it's built into a 1 element list
+            if isinstance(data, list):
+                data = data[0]
+            data = [int(x) for x in data.split(',')]
```

Comment:
Need to catch any decode errors and instead raise `serializers.ValidationError`. Also, we should probably guarantee positive ints.